frontend/components/TeamSection/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Heading, GradientText } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const TeamGroupSection = ({ data }) => {
+  const {
+    title,
+    userOne,
+    userTwo,
+    userTitle,
+    description,
+    userOneImg,
+    userTwoImg
+  } = data;
+
+  // const userOneData = {
+  //   image: userOneImg,
+  //   title: userOne,
+  //   subtitle: userTitle,
+  //   description: description
+  // };
+  // const userTwoData = {
+  //   image: userTwoImg,
+  //   title: userTwo,
+  //   subtitle: userTitle,
+  //   description: description
+  // };
+
+  const usersData = [
+    {
+      image: userOneImg,
+      title: userOne,
+      subtitle: userTitle,
+      description: description
+    },
+    {
+      image: userTwoImg,
+      title: userTwo,
+      subtitle: userTitle,
+      description: description
+    }
+  ];
+
+  const cardData = usersData.map((userData, index) => {
+    return (
+      <div>
+        <PersonCard
+          cardDetails={usersData[index]}
+          rounded={true}
+          border={true}
```

Comment:
this prop is not even in the component you should know because it doesn't show a border right ? 😄 and you still nee to ad margin between the 2 cards (not padding to show the border).

Reply:
I was trying to figure how border works :smile: and i forget to remove it after.